scripts/utils.sh

```diff
@@ -67,19 +68,34 @@ function exportPaths () {
     echo "→ KEYCLOAK_BINARIES_PATH=$KEYCLOAK_BINARIES_PATH"
     echo "→ EIP_OPENMRS_ORTHANC_ROUTES_PATH=$EIP_OPENMRS_ORTHANC_ROUTES_PATH"
     echo "→ ORTHANC_CONFIG_PATH=$ORTHANC_CONFIG_PATH"
-
+    echo "→ BAHMNI_OPENMRS_MODULES_PATH=$BAHMNI_OPENMRS_MODULES_PATH"
+    echo "→ BAHMNI_CONFIG_OVERRIDE_PATH=$BAHMNI_CONFIG_OVERRIDE_PATH"
 }
 
 function setDockerComposeCLIOptions () {
     # Parse 'docker-compose-files.txt' to get the list of Docker Compose files to run
     dockerComposeFiles=$(cat docker-compose-files.txt)
+
+    if [ "$RUN_WITH_BAHMNI_EMR" == "true" ]; then
+        export ENABLE_SSO=false
+    fi
+
     for file in ${dockerComposeFiles}
     do
         if [ "$ENABLE_SSO" != "true" ]; then
             if [[ "$file" == *"-sso.yml" || "$file" == "docker-compose-keycloak.yml" ]]; then
                 continue
             fi
         fi
+
+        if [[ "$file" == *"-bahmniemr.yml" && "$RUN_WITH_BAHMNI_EMR" == "false" ]]; then
+            continue
+        fi
+
+        if [[ "$file" == *"-openmrs.yml" && "$RUN_WITH_BAHMNI_EMR" == "true" ]]; then
+            continue
+        fi
```

Comment:
This will break `start.sh` when run with SSO enabled.

Reply:
There's logic to fail-over to using O3 above this if SSO is enabled.